LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Juan Pedro Villa Isaza
+Copyright (c) 2017 Juan Pedro Villa Isaza, (c) 2022 Mercury Technologies, Inc
```

Comment:
I'm not sure how copyright works for abandon or forked open source projects, TBH. Is this an educated guess on your part or is this legit how it's done?